terraform/dynamodb.tf

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+resource "aws_dynamodb_table" "users_table" {
```

Comment:
So i guess my question is why is there a `dyanmodb.tf` file and then there is something similar in `terraform/global/dynamodb/main.tf`

What are the differences?

Reply:
The global dynamo is the Terraform locks table for deployments.  This table represents the constructs an application level, users object table for consumption by the application.